kafka-ui-react-app/src/components/Brokers/Broker/Configs/Configs.tsx

```diff
@@ -34,14 +34,12 @@ const Configs: React.FC = () => {
 
   const getData = () => {
     return data
-      .filter(
-        (item) =>
-          item.name.toLocaleLowerCase().indexOf(keyword.toLocaleLowerCase()) >
-          -1
-      )
+      .filter((item) => {
+        const nameMatch = item.name.toLocaleLowerCase().includes(keyword);
+        return nameMatch ? true : item.value && item.value.includes(keyword); // try to match the keyword on any of the item.value elements when nameMatch fails but item.value exists
```

Comment:
@malavmevada shouldn't we cast item value to lower case as well?

Reply:
@Haarolean Actually, we shouldn't convert the 'item' values to lowercase because if you observe, all the values under the 'name' key are in lowercase strings. However, for the 'item' key, we can have values in uppercase, lowercase, null, and numbers. Due to this diversity, I haven't changed the 'item' values to lowercase.

Reply:
@malavmevada sorry for the delay. Isn't this the point? Let's say I have some setting but in order to find it but its value I have to remember the exact case-sensitive value?

Reply:
Yes you are absolutely right. Sorry for the mistake. I got your point and I have tested it and yeah it's working fine.